app/assets/stylesheets/all/users.css.scss

```diff
@@ -3,12 +3,17 @@
     margin-bottom: 10px;
   }
 
+  td {
+    font-size: 15px;
+  }
+
   td.name {
-    width: 100px;
+    width: 150px;
   }
 
   td.value {
     font-weight: bold;
+    font-family: monospace;
```

Comment:
I originally had, but there's no actual way to use the defined variables in other files, then forgot to un-define the variable since I couldn't use it. If I had more time, I'd have tried to refactor the variables out into their own file that could be included in the other files and then fix all the places that manually re-define font-family stacks, but that seems like an issue for the backlog since it's a little more extensive.